plugins/woocommerce-admin/client/marketplace/components/product-list-content/product-list-content.tsx

```diff
@@ -4,11 +4,32 @@
 import './product-list-content.scss';
 import ProductCard from '../product-card/product-card';
 import { Product } from '../product-list/types';
+import { appendURLParams } from '../../utils/functions';
+import { getAdminSetting } from '../../../utils/admin-settings';
 
 export default function ProductListContent( props: {
 	products: Product[];
 } ): JSX.Element {
-	const { products } = props;
+	const wccomHelperSettings = getAdminSetting( 'wccomHelper', {} );
+
+	const products = props.products.map( ( product ) => {
```

Comment:
I see we map here and then map again further down. Not critical, but I imagine if we improved the `appendURLParams()` method a bit, we could do this as part of the map method later in this file without too much messiness.

That said, I don't think that's going to make any noticeable difference on performance.

Reply:
Looks good to me! I think this is the best approach over all.

Reply:
OK, I have pushed up this change and it's ready for a review.